app/helpers/template_helpers.py

```diff
@@ -44,6 +44,42 @@ def get_page_header_context(language, theme):
 
 
 def get_footer_context(language_code, static_content_urls, sign_out_url, theme):
+    if theme == "census-nisra":
+        items_list = [
+            {
+                "text": lazy_gettext("Help"),
+                "url": static_content_urls["help"],
+                "target": "_blank",
+            },
+            {
+                "text": lazy_gettext("Contact us"),
+                "url": static_content_urls["contact_us"],
+                "target": "_blank",
+            },
+        ]
+    else:
+        items_list = [
+            {
+                "text": lazy_gettext("Help"),
+                "url": static_content_urls["help"],
+                "target": "_blank",
+            },
+            {
+                "text": lazy_gettext("Contact us"),
+                "url": static_content_urls["contact_us"],
+                "target": "_blank",
+            },
+            {
+                "text": lazy_gettext("Languages"),
+                "url": static_content_urls["languages"],
+                "target": "_blank",
+            },
+            {
+                "text": lazy_gettext("BSL and audio videos"),
+                "url": static_content_urls["bsl_and_audio_videos"],
+                "target": "_blank",
+            },
+        ]
```

Comment:
Since the first two is always there, this can be simplified:
```py
    items_list = [
        {
            "text": lazy_gettext("Help"),
            "url": static_content_urls["help"],
            "target": "_blank",
        },
        {
            "text": lazy_gettext("Contact us"),
            "url": static_content_urls["contact_us"],
            "target": "_blank",
        },
    ]
    if theme != "census-nisra":
        items_list.extend(
            [
                {
                    "text": lazy_gettext("Languages"),
                    "url": static_content_urls["languages"],
                    "target": "_blank",
                },
                {
                    "text": lazy_gettext("BSL and audio videos"),
                    "url": static_content_urls["bsl_and_audio_videos"],
                    "target": "_blank",
                },
            ]
        )
```

Reply:
Done, thanks!